src/Configuration/CliOptions.cs

```diff
@@ -71,7 +71,23 @@ public static (PlcSimulation PlcSimulationInstance, List<string> ExtraArgs) Init
                         throw new OptionException("The max OPC string length must be larger than 0.", "opcmaxstringlen");
                     }
                 }
-            },
+            },
+
+            // OTLP Exporter Configuration
+            { "otlpen|otlpenabled", $"enables the otlp exporter.\nDefault: {config.OtlpEnabled}", (s) => config.OtlpEnabled = s != null },
+            { "otlpep|otlpendpoint=", $"the endpoint URI to which the otlp exporter is going to send traces.\nDefault: {config.OtlpEndpointUri}", (s) => config.OtlpEndpointUri = s },
+            { "otlpei|otlpexportinterval=", $"the interval to export traces in seconds.\nDefault: {config.OtlpExportInterval}", (int i) => {
+                    if (i > 0)
```

Comment:
I suggest to add another option to specify the export protocol (grpc or protobuf)

Reply:
Done, txs